src/Http/Controllers/ElectronicSignatureWebhookController.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace AlNutile\DocusignDriver\Http\Controllers;
+
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Request;
+
+class ElectronicSignatureWebhookController
+{
+    public function handleWebhook(Request $request): JsonResponse
+    {
+        $webhook = $request->all();
+
+        if ($webhook['event'] !== 'recipient-completed') {
+            return new JsonResponse([], 200);
+        }
+
+        \Log::info($webhook);
```

Comment:
ok so make a Job in the next PR
and save the payload to the folder tests/fixtures/webhook.json so I can see what the incoming payload we want looks like so later I can write code and tests